create-jira-release-ticket/action.yml

```diff
@@ -88,27 +81,48 @@ runs:
       env:
         JIRA_USER: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_USER }}
         JIRA_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}
+        JIRA_PROD_URL: "https://sonarsource.atlassian.net/"
+        JIRA_SANDBOX_URL: "https://sonarsource-sandbox-608.atlassian.net/"
       run: |
-        SANDBOX_FLAG=""
-        if [[ "${{ inputs.use_sandbox }}" == "true" ]]; then
-          SANDBOX_FLAG="--use-sandbox"
+        # Determine project key from input or environment variable
+        PROJECT_KEY="${{ inputs.jira-project-key }}"
+        if [[ -z "$PROJECT_KEY" ]]; then
+          PROJECT_KEY="${JIRA_PROJECT_KEY:-}"
+        fi
+
+        # Fail if no project key is provided
+        if [[ -z "$PROJECT_KEY" ]]; then
+          echo "Error: No Jira project key provided. Please set either the 'jira-project-key' input or the 'JIRA_PROJECT_KEY' environment variable."
+          exit 1
+        fi
+
+        # Determine release URL from input or environment variable
+        RELEASE_URL="${{ inputs.jira-release-url }}"
+        if [[ -z "$RELEASE_URL" ]]; then
+          RELEASE_URL="${JIRA_RELEASE_URL:-}"
+        fi
```

Comment:
Same comment as above, and as a fallback, you can use the new `action get-jira-release-notes`